src/Sulu/Bundle/ContactBundle/Content/Types/SingleAccountSelection.php

```diff
@@ -70,9 +57,9 @@ public function write(
         $languageCode,
         $segmentKey
     ) {
-        $account = $property->getValue();
-        if (null != $account) {
-            $node->setProperty($property->getName(), $account['id']);
+        $value = $property->getValue();
+        if (null != $value) {
+            $node->setProperty($property->getName(), is_array($value) ? $value['id'] : $value);
```

Comment:
I think we should deprecate the $value['id'] and throw a deprecation message here as it should not longer be send this way to the backend?

Reply:
sounds good 👍